tests/fixtures/sample_lib/modules/sample_task/sample_implementation.py

```diff
@@ -69,6 +70,7 @@ def save(self, model_path):
     def train(
         cls,
         training_data: DataStream[SampleTrainingType],
+        union_list: Union[List[str], List[int]] = None,
```

Comment:
I needed another module that has the same name argument for unions of lists

Reply:
I'm not really following this exactly, but would the deduced request here be `union_list.union_list.values` to get at the actual values?

Also, if this can be `None`, it should be marked `Optional`